### ISSUE TITLE
--sub-font='fontname' is ignored

### ISSUE THREAD

**Issue by Silejonu** [celluloid-player/celluloid#507]
Overview Description:
The option `--sub-font='fontname'` is ignored both in the extra MPV options and in the mpv config file.

Steps to Reproduce:
1. add `--sub-font='fontname'` to mpv.conf or the extra MPV options

Actual Results:
The subtitles font is the default Celluloid one.

Expected Results:
The chosen font for subtitles should be shown.

Version:
0.18 (running Fedora Workstation 31)

Additional Information:
Adding the option to standalone mpv via the command line or the config file works as expected.

**Comment by gnome-mpv** [comment]
Are you using Flatpak?

**Comment by Silejonu** [comment]
No, I am not using Flatpak.
Just to be sure, I tested it under Arch Linux, and actually, the issue is not what I thought it was. The `--sub-font` option is working fine. The issue is that commands containing `"` or `'` in the extra MPV options fail every time and return "Failed to apply one or more MPV options" upon opening a file. They do work fine if they are written in a config file, though (I probably made a typo when I tested it earlier, this would explain why I thought it didn't work in config files). Passing the commands with `"` or `'` in standalone mpv work just fine.
Should I open another ticket for this or just edit my original message?
